Week3/Homework/Js-exercises/dogAgeCalculator.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+let calculateDogAge = function(age) {
+  let dogAge = age * 7;
+
+  console.log("Your doggie is " + dogAge + " years old in dog age!");
```

Comment:
I think it will be better if you returned the string from here rather than logging it inside the function.